src/t_stream.c

```diff
@@ -3704,7 +3662,9 @@ void xinfoReplyWithStreamInfo(client *c, stream *s) {
     addReplyBulkCString(c,"max-deleted-entry-id");
     addReplyStreamID(c,&s->max_deleted_entry_id);
     addReplyBulkCString(c,"entries-added");
-    addReplyLongLong(c,s->entries_added);
+    addReplyUnsignedLongLong(c,s->entries_added);
+    addReplyBulkCString(c,"recorded-first-entry-id");
+    addReplyStreamID(c,&s->first_id);
```

Comment:
is this the first-ever ID? or the current first? i.e. if I add 100-0 and 100-1 and then XDEL 100-0 what will be s->first_id?

if it's 100-1 i think it should just be called "first-id" and not "recorded-first-entry-id"

Reply:
It would be 100-1, and it should be call "first-id" in the info but we already have another "first-id" there (the two are supposed to be identical).